src/pages/gen2/build-a-backend/data/data-modeling/identifiers/index.mdx

```diff
@@ -45,11 +45,12 @@ const schema = a.schema({
     title: a.string(),
     completed: a.boolean(),
   }).identifier(['todoId'])
+  .authorization([a.allow.public()]),
 });
 
 const client = generateClient<Schema>();
 
-const todo = await client.models.Todo.create({ todoId: 'MyUniqueTodoId', title: 'Buy Milk', completed: false });
+ const { data: todo, errors } = await client.models.Todo.create({ todoId: 'MyUniqueTodoId', title: 'Buy Milk', completed: false });
```

Comment:
do we want to keep the unused variable `errors` in the example?

Reply:
yeah, just for visibility that it's available.